python/ql/src/experimental/dataflow/internal/DataFlowPublic.qll

```diff
@@ -138,6 +145,58 @@ class BarrierGuard extends Expr {
 /**
  * A reference contained in an object. This is either a field or a property.
  */
-class Content extends string {
-  Content() { this = "Content" }
+newtype TContent =
+  /** An element of a list. */
+  TListElementContent() or
+  /** An element of a set. */
+  TSetElementContent() or
+  /** An element of a tuple at a specifik index. */
+  TTupleElementContent(int index) { exists(IntegerLiteral lit | lit.getValue() = index) } or
```

Comment:
Can be replaced with `exists(any(TupleNode tn).getElement(index))` to make it smaller.

Reply:
Neat, done.